test/gh-issues/bug-205.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const { test } = require('tap')
+const boot = require('../..')
+
+test('should print the time tree', (t) => {
+  t.plan(2)
+  const app = boot()
+
+  app.use(function first (instance, opts, cb) {
+    const out = instance.prettyPrint().split('\n')
+    t.equal(out[0], 'bound root undefined ms')
```

Comment:
undefined?

Reply:
yes, what do you prefer? 0 is misleading.  at that point there is no diff. So what should be printed?

Reply:
For something that says "not loaded yet", Maybe `-1`?

Reply:
k, will do. Currently writing unit tests.